developer/pages-router/localization.mdx

```diff
@@ -6,105 +6,114 @@ description: "This guide walks you through setting up localization for your Make
 import PagesCatchAllLocalizedExample from "/snippets/pages-router/catch-all-localized.mdx";
 import NextConfigLocalizedExample from "/snippets/pages-router/next-config-localized.mdx";
 import NextConfigDomainLocalizedExample from "/snippets/pages-router/next-config-domain-localized.mdx";
+import PagesAppLocalized from "/snippets/pages-router/pages-app-localized.mdx";
+
+If you don't already have a Next.js project using Pages Router set up with Makeswift, follow the [Pages Router Installation](/pages-router/installation) guide.
 
 <Info>
   The feature is deeply integrated with [Next.js' internationalization
   features](https://nextjs.org/docs/pages/building-your-application/routing/internationalization).
 </Info>
 
-## Getting started
-
-<Steps>
+## Add locales in the Visual Builder
```

Comment:
Given that [we consider locales to be a host-driven feature](https://makeswifthq.slack.com/archives/C088ZQEUH42/p1739374577266239?thread_ts=1739330498.350359&cid=C088ZQEUH42), I wonder if we should swap the order of these sections and start with adding the locales to the Next.js config. At some point, we'll have a host-to-builder locale sync that will eliminate step 2.